jerry-core/api/jerry.c

```diff
@@ -782,6 +783,23 @@ jerry_value_is_undefined (const jerry_value_t value) /**< api value */
   return ecma_is_value_undefined (value);
 } /* jerry_value_is_undefined */
 
+/**
+ * Perform strict equality comparison on the given operands.
+ *
+ * See also ECMA-262 v5.1, 11.9.6
+ *
+ * @return true  - if the two operands are strictly equal,
+ *         false - otherwise.
+ */
+bool
+jerry_value_strict_equal (const jerry_value_t lhs, /**< first operand */
+                          const jerry_value_t rhs) /**< second operand */
+{
+  jerry_assert_api_available ();
+
```

Comment:
If either of the arguments is an error reference, is it able to cause any side effect? Maybe a test would be great for it.

Reply:
Also if this is a special case the documentation should mention it.